### PR TITLE
all.xml: Add message/command ranges for other dialects

### PR DESCRIPTION
**Pull request by hamishwillee**
This adds additional range information to all.xml. I'd rather have this in the files themselves, but good to have an overview.

This is from https://mavlink.io/en/guide/define_xml_element.html and by inspection.

@peterbarker OK?

### CODE REVIEW

File: message_definitions/v1.0/all.xml [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
   <include>ardupilotmega.xml</include>
   <!-- ASLUAV.xml range of IDs:
     messages: 8000 - 8999
     commands: 40001 - 41999
   -->
   <include>ASLUAV.xml</include>
+  <!-- common.xml range of IDs:
+    messages: 300 - 10000
+    commands: 0 - 39999
+  -->
```
Comment (peterbarker) [inline message_definitions/v1.0/all.xml:16]: Hmmm.  We probably need a comment in here somewhere about this being the general location of things.  Several things have made their way from other files into common....

**Review by peterbarker — APPROVED** [review]
Nice to have something in here, at least.

File: message_definitions/v1.0/all.xml (hunk 1 above)
Comment (hamishwillee) [reply message_definitions/v1.0/all.xml:16]: Good point
```suggestion
  <!-- common.xml range of IDs:
    messages: 300 - 10000
    commands: 0 - 39999
  
    Note: entities imported from other dialects may fall outside these ranges.    
  -->
```

File: message_definitions/v1.0/all.xml [hunk 2]
```diff
@@ -45,8 +63,8 @@
     commands: 52000 - 52099
   -->
   <!--Next range to allocate range of IDs:
-    messages: 52100 - ? < 60000
-    commands: 52100 - ? < 60000
+    messages: 52100 - 52100 < 60000
+    commands: 52100 - 52199 < 60000
```
Comment (hamishwillee) [inline message_definitions/v1.0/all.xml:67]:
```suggestion
    messages: 52100 - 52100 (< 60000)
    commands: 52100 - 52199 (< 60000)
```

File: message_definitions/v1.0/all.xml [hunk 3]
```diff
@@ -1,13 +1,27 @@
...
   <include>ardupilotmega.xml</include>
   <!-- ASLUAV.xml range of IDs:
     messages: 8000 - 8999
     commands: 40001 - 41999
   -->
   <include>ASLUAV.xml</include>
+  <!-- common.xml range of IDs:
+    messages: 300 - 10000
+    commands: 0 - 39999
+
```
Comment (hamishwillee) [inline message_definitions/v1.0/all.xml:16]:
```suggestion
```

**Comment by hamishwillee** [comment]
Thanks. I've added a note, and will merge on passing.

**Pull request merged**